config/infrastructure/aws/README.md

```diff
@@ -0,0 +1,56 @@
+# Deploying GCFroms on Amazon Web Services (AWS)
+
+:warning: This is not a fully featured production environment and aims to provide an accessible overview of the service.
+
+This document describes how to deploy and operate a **reference implementation** of the COVID Alert Portal.
+
+There should be an illustration of the Covid Shield infrastructure deployed on AWS right here (TODO)
+
+At a glance, health care professionals (on the left) interact with a web portal, and mobile app users (on the right) interact with the diagnosis key retrieval and submission services.
+
+## IT Service Requirements
+
+(TODO)
+
+| Service                  | AWS product offering                                                   |
+| ------------------------ | ---------------------------------------------------------------------- |
+| Serverless compute       | [Fargate](https://aws.amazon.com/fargate/)                             |
+| Container registry       | [Elastic Container Registry](https://aws.amazon.com/ecr/)              |
+| Domain name services     | [Route 53](https://aws.amazon.com/route53/)                            |
+| TLS certificates         | [Certificate Manager](https://aws.amazon.com/certificate-manager/)     |
+| Load balancing           | [Elastic Load Balancing](https://aws.amazon.com/elasticloadbalancing/) |
+| Content delivery network | [CloudFront](https://aws.amazon.com/cloudfront/)                       |
+| Web application firewall | [WAF](https://aws.amazon.com/waf/)                                     |
+
+## Deploying COVID Alert Portal
```

Comment:
I'm in the process of removing some references to the c19-benefits-node too, so we can rename these in other PRs :-)